packages/ui/src/components/sign-in/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from '~/primitives/button';
+import * as Card from '~/primitives/card';
+import * as Connection from '~/primitives/connection';
+import * as Field from '~/primitives/field';
+import { Seperator } from '~/primitives/seperator';
+
+export function SignIn() {
+  return (
+    <Card.Root>
+      <Card.Content>
+        <Card.Header>
+          <Card.Title>Sign in to Acme Co</Card.Title>
+          <Card.Description>Welcome back! Please sign in to continue</Card.Description>
+        </Card.Header>
+        <Card.Body>
+          <Connection.Root>
+            <Connection.Button>Google</Connection.Button>
+            <Connection.Button>GitHub</Connection.Button>
+          </Connection.Root>
+          <Seperator>or</Seperator>
+          <Field.Root>
+            <Field.Label>Email address</Field.Label>
+            <Field.Input />
+            <Field.Message intent='error'>Identifier is invalid.</Field.Message>
+          </Field.Root>
+          <Button>Continue</Button>
+        </Card.Body>
+      </Card.Content>
+      <Card.Footer>
+        <Card.FooterAction>
+          <Card.FooterActionText>
+            Don&apos;t have an account? <Card.FooterActionLink href='/sign-up'>Sign up</Card.FooterActionLink>
```

Comment:
we should probably consider i18n even from this early stage. it doesn't have to be a blocker for this PR but we should consider it a fast'ish follow imo

Reply:
Adding a ticket to Linear for follow-up 🫡